src/AppBundle/Entity/Post.php

```diff
@@ -179,4 +192,38 @@ public function setSummary($summary)
     {
         $this->summary = $summary;
     }
+
+    /**
+     * Add tag
+     *
+     * @param Tag $tag
+     *
+     * @return Post
+     */
+    public function addTag(Tag $tag)
+    {
+        $this->tags[] = $tag;
```

Comment:
Certainly checking for duplicates or setting the relation for the other side avoid lots of errors but it can be a very heavy task for large collection.